redis.conf

```diff
@@ -286,9 +286,15 @@ pidfile /var/run/redis_6379.pid
 # warning (only very important / critical messages are logged)
 loglevel notice
 
-# Specify the log file name. Also the empty string can be used to force
-# Redis to log on the standard output. Note that if you use standard
-# output for logging but daemonize, logs will be sent to /dev/null
+# To enable raw logging, set this to 'yes'. By default the logging on
+# the 'logfile' contains timestamp, pid and other information, which this
+# option can disable. Helpful when logging to stdout/stderr on systemd.
+lograw no
+
+# Specify the log file name. Setting it to 'none' will avoid using any
```

Comment:
+1 since at least with systemd, [it's trivial](https://serverfault.com/questions/838574/how-to-silent-a-systemd-service) to `/dev/null` stdout if needed.

Reply:
Hmm, that was some time ago so cannot recall the original thinking. Perhaps because the code contemplated that case, it seemed better to avoid wasting work. If this is not desired, I can take this one out.

Reply:
Ah I recalled now. So if you enable syslog logging, then you probably do not want to log into a file, which seems like a waste, even if it is redirected to /dev/null.

Reply:
I don't really like the idea of a constant "none" string here and adding and extra `disable-logfile` config seems overkill. `/dev/null` sounds like a good compromise. I'd take this out of this PR.